packages/api/src/shared/helpers/DateHelper.ts

```diff
@@ -68,3 +68,5 @@ export const getShortISODate = (date: Date) => date.toISOString().substring(0, 1
 export const shortISORegExp = new RegExp(/\d{4}-[01]\d-[0-3]\d/);
 
 export const shortISOPeriodRegExp = new RegExp(/\d{4}-[01]\d-[0-3]\d\/\d{4}-[01]\d-[0-3]\d/);
+
+export const addDaysToDate = (date, nbOfDays = 1) => new Date(new Date(date).setDate(date.getDate() + nbOfDays));
```

Comment:
pourquoi ça ne suffit pas avec un constructeur en moins ?
```suggestion
export const addDaysToDate = (date, nbOfDays = 1) => new Date(date).setDate(date.getDate() + nbOfDays);
```

Reply:
Parce que setDate ne renvoi pas une date mais un entier représentant le nombre de milliseconde.

C'est soit ça soit on écrit comme ceci :

```js
const newDate = new Date(date);
newDate.setDate(newDate.getDate());
return newDate
```

Reply:
ok je pensais pas ; mais du coup va pour ça

Reply:
oui et c'est nul parce que 1) ça modifie quand même la date 2) ça renvoi pas la date :/